src/P3_terminal_velocity.jl

```diff
@@ -1,141 +1,59 @@
+
 """
-    ice_particle_terminal_velocity(state, D, Chen2022, ρₐ, use_aspect_ratio)
-Returns the terminal velocity of a single ice particle as a function
-    of its size (maximum dimension, D) using the Chen 2022 parametrization.
+    ice_particle_terminal_velocity(state, velocity, ρₐ; use_aspect_ratio)
```

Comment:
Same as above. I don't understand why we need two options, instead of just having the function defined and then using it to obtain a value.

Reply:
See above, but can abandon this if it seems too confusing/not useful/hard to maintain.